### ISSUE TITLE
channel_priority=strict will result in package conflicts during installation

### ISSUE THREAD

**Issue by BerginJay** [DOI-USGS/ISIS3#5158]
**ISIS version(s) affected**: 4.1.0, 7.1.0, 7.2.0 (maybe more, I only tested these versions)

**Description**  
<!-- A clear and concise description of the problem. -->
An error occurred during installation using conda:

- Using `conda install`

```bash
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                             
UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/osx-64::__osx==12.6.1=0
  - isis -> pcl -> __osx[version='>=10.12']

Your installed version is: 12.6.1

Note that strict channel priority may have removed packages required for satisfiability.
```

- Using `mamba install`:

```bash
# Version 4.1.0
Could not solve for environment specs
Encountered problems while solving:
  - package isis-4.1.0-0 requires geos >=3.7,<3.8, but none of the providers can be installed

The environment can't be solved, aborting the operation


# Version 7.1.0
Could not solve for environment specs
Encountered problems while solving:
  - package isis-7.1.0-0 requires geos >=3.7,<3.8.0a0, but none of the providers can be installed

The environment can't be solved, aborting the operation


# Version 7.2.0
Could not solve for environment specs
Encountered problems while solving:
  - package isis-7.2.0-np124_0 requires embree >=2.17,<2.18.0a0, but none of the providers can be installed

The environment can't be solved, aborting the operation
```

**How to reproduce**  
<!-- Code and/or config needed to reproduce the problem. -->
```bash
conda create -n isis
conda config --env --add channels conda-forge
conda config --env --add channels usgs-astrogeology

# This is the key
conda config --env --set channel_priority strict

mamba install -c usgs-astrogeology isis=7.2.0  
# You will get a similar error with conda, but it will take more time to solve the environment
```


**Possible Solution**  
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->
`channel_priority=strict` would cause such error.
The point is that when someone using conda sets `channel_priority=strict` globally and forgets about it, they may be confused by such an error reported.
Recommended to set `channel_priority=flexible` for isis environment manually during installation.

**Additional context**  
<!-- Optional: any other context about the problem: log messages, screenshots, etc. -->
Related `PR` is on the way.

**Comment by BerginJay** [comment]
Fixed in #5159.

**Comment by steo85it** [comment]
Thanks, but even with 
```
channels:
  - usgs-astrogeology
  - conda-forge
  - defaults
channel_priority: flexible
```
I still get an infinite loop of
```
(isis) >> conda update -c usgs-astrogeology isis
Collecting package metadata (current_repodata.json): done
Solving environment: / 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed with repodata from current_repodata.json, will retry with next repodata source.
```
Are you sure #5159 fixes the issue?
